### PR TITLE
Chapter 16 - First Part Summary

### PR DESCRIPTION
**Pull request by AngelFelizR**

### CODE REVIEW

File: 16_interpretable-machine-learning.Rmd [hunk 1]
```diff
@@ -1,13 +1,464 @@
...
+- **Local Interpretation** becomes very important when the effect of most influential model's features doesn't the biggest influence for a particular observation as it helps us to understand what **features are influencing the predicted response** for a given observation or an small group of observations.
+  - Local interpretable model-agnostic explanations (LIME)
+  - Shapley values
+  - Localized step-wise procedures
+
+## Setting enviroment
+
+### Loading libraries
+
+```{r libraries, message=FALSE, echo=TRUE, results='hide'}
```
Comment (jonthegeek) [inline 16_interpretable-machine-learning.Rmd]:
```suggestion
```{r 16-libraries, message=FALSE, echo=TRUE, results='hide'}
```

**Review by jonthegeek — CHANGES_REQUESTED** [review]
Need to make the chunk name unique for it to build.

**Review by jonthegeek — DISMISSED** [review]
Looks good as long as it builds!

File: 16_interpretable-machine-learning.Rmd (hunk 1 above)
Comment (AngelFelizR) [reply 16_interpretable-machine-learning.Rmd]: Thanks, I didn't know that

**Review by jonthegeek — APPROVED** [review]
Thanks!

**Pull request merged**